paasta_tools/kubernetes_tools.py

```diff
@@ -1432,6 +1459,41 @@ def get_boto_volume(self) -> Optional[V1Volume]:
         )
         return volume
 
+    def get_crypto_volume(self) -> Optional[V1Volume]:
+        required_crypto_keys = get_crypto_keys_from_config(
+            self.config_dict.get("crypto_keys", {})
+        )
```

Comment:
thoughts on moving `get_crypto_keys_from_config` into this class so that this looks like:

```suggestion
        required_crypto_keys = self.get_crypto_keys_from_config()
```

and `get_crypto_keys_from_config` then looks like:
```python
def get_crypto_keys_from_config() -> List[str]:
    encryption_keys = self.config_dict.get("crypto_keys", {}).get("encrypt", [])
    decryption_keys = self.config_dict.get("crypto_keys", {}).get("decrypt", [])

    return [
        *(f"public/{key}" for key in encryption_keys),
        *(f"private/{key}" for key in decryption_keys),
    ]
```

it seems unlikely that we'll want to call get_crypto_keys_from_config outside the context of this class, so we might as well use the state that the class already holds rather than making a free function and having to split the dict traversal up into callee and caller portions